kani-compiler/src/codegen_cprover_gotoc/codegen/block.rs

```diff
@@ -72,3 +73,25 @@ impl<'tcx> GotocCtx<'tcx> {
         }
     }
 }
+
```

Comment:
Can you replicate the explanation in the PR's description here? Just want to make sure we don't wonder in the future why we're not using the version available from the compiler :)

Reply:
sure